setup_sunvdm.ps1

```diff
@@ -1,10 +1,12 @@
 Import-Module WindowsDisplayManager
 
+Set-ExecutionPolicy -Scope Process -ExecutionPolicy Bypass
+
 # Snapshot the current display state so we can restore it after the session.
 #
-if ($args.Length -lt 4)
+if ($args.Length -lt 5)
 {
-    Throw "Incorrect number of args: should pass WIDTH HEIGHT REFRESH_RATE HDR{0|1}"
+    Throw "Incorrect number of args: should pass WIDTH HEIGHT REFRESH_RATE HDR{0|1} VDD_NAME"
 }
```

Comment:
I'd like to keep a standard where updates to this script won't break existing installs - can we make VDD_NAME optional, and default it to the previous "root\iddsampledriver"?

Reply:
VDD_NAME shouldn't be `root\iddsampledriver` but equals to the actual driver name displayed in the Device Manager.
So it should be either `IddSampleDriver Device HDR` (for Windows 11 driver) or `Virtual Display Driver by MTT` (for Windows 10 driver)

I modified how to enable the virtual display because of cold reboot issues: (see the end of [this issue](https://github.com/LeGeRyChEeSe/Sunshine-AIO/issues/4))
`Get-PnpDevice -FriendlyName $vdd_name | Enable-PnpDevice -Confirm:$false`

I can surely make this new variable optional, but users running old scripts will still get this issue with cold reboots.

So imo if a user wants to update the scripts, so he has to modify his setup to run without issues.

Reply:
I didn't realize it had to be a different format - in that case, I would still prefer to have a sane default ("IddSampleDriver Device HDR" should be fine for default then?), and have VDD as an optional parameter - alternatively if you can automatically determine it in the script that would be awesome :)